src/site/content/en/reliable/codelab-http-cache/codelab-http-cache.md

```diff
@@ -159,8 +159,7 @@ working through [this codelab](/codelab-explore-network-panel).
 With the modifications to the static file server in place, you can check to make
 sure that the right headers are being set by previewing the live app with the DevTools Network panel open.
 
-- To preview the site, mouse over the editor, press the **App** button, then the
-  **Show** button.
+{% Instruction 'preview' %}
 
 - Customize the columns that are
 displayed in the Network panel to include the information that is most relevant, by right-clicking in
```

Comment:
```suggestion
displayed in the **Network** pane to include the information that is most relevant by right-clicking in
```
Per DSG: https://developers.google.com/style/ui-elements#term-windows

Reply:
ah good catch. I think the DevTools docs themselves refer to these as "panels" so we might keep it to stay inline with them.
https://developers.google.com/web/tools/chrome-devtools/network/#overview

Reply:
OK, cool--as long as we're consistent. I'll add a note to our writing handbook.